src/pages/talks/index.js

```diff
@@ -1,15 +1,122 @@
 import React from 'react';
 import Helmet from 'react-helmet';
+import withStyles from 'react-jss';
+import cx from 'classnames';
 
-import { Header, Navigation } from '../../components';
+import imagePatternDots from '../../assets/images/patterns/dots.png';
+import { Header, Heading, Drops, Talk, Footer } from '../../components';
+import { Contact, SignUp } from '../../containers';
+import talks from '../../data/talks.json';
 
-const TalksPage = () => (
+const styles = theme => ({
+  root: {
+    display: 'flex',
+    minHeight: '100vh',
+    flexDirection: 'column'
+  },
+
+  // MAIN
+  main: {
+    flex: '1'
+  },
+  mainContainer: {
+    padding: [20, 0],
+    '& .row': {
+      margin: 0
+    },
+    '& .col-xs-12': {
+      padding: 0
+    }
+  },
+  componentHeading: {
```

Comment:
"mainHeading". Remember this styling section has the name "main" so most if not all classes should start with main for readability. Apply this everywhere else, please.

Reply:
ok, I make the change, these conventions i take that as I do for everything, with reference to how you declare it in other places